libs/dh/admin/feature-user-roles/src/lib/create/dh-create-user-role-modal.component.ts

```diff
@@ -136,17 +133,15 @@ export class DhCreateUserRoleModalComponent implements OnInit, AfterViewInit {
     this.userRoleForm.controls.eicFunction.valueChanges
```

Comment:
Suggestion: This can be done in another PR.
Convert valueChange to signal, and run refretch in an effect